WebRoot/js/zimbraMail/prefs/view/ZmFilterPage.js

```diff
@@ -36,6 +36,7 @@
  */
 ZmFilterPage = function(parent, section, controller) {
 	ZmPreferencesPage.apply(this, arguments);
+	this.addClassName("ZmFilterPage");
```

Comment:
it's ok I thought we are supporting passing any number of arguments as part of params
ignore this comment